### DISCUSSION TITLE
The mass of the primary star

### DISCUSSION THREAD

**Discussion by hulin-Lee** [TeamCOMPAS/COMPAS#933] [Q&A] [answered]
When I used the command of  ./COMPAS -n 10000 to generate 10000 binaries, I fond that the program produced 55 binaries, where the primary masses are less than 5 solar mass. Moreover, mass ratios of these binaries are all equal to one. Then, I tried to set the lower limit of the primary mass equal to 5 solar mass explicitly, but still found the same phenomenon. I wonder is there something wrong with the way I use COMPAS. Thanks a lot!

**Comment by SimonStevenson** [comment, marked as answer]
Hi @hulin-Lee , I think these are probably systems that are born with very small separations and RLOF at birth, leading them to have their masses equalised. There is a flag in the SystemParameters file (MASSES_EQUILIBRATED_AT_BIRTH, see [here](https://compas.readthedocs.io/en/latest/pages/User%20guide/COMPAS%20output/standard-logfiles-default-record-specifications-BSE-sysparms.html)) that tells you if this has occurred. This step happens after the initial sampling, so if you draw a binary with a primary of 6 MSun and a secondary of 3 MSun, if the masses are equilibrated it would produce a binary with both components being 4.5Msun. You could just manually filter these out, or you could try disabling the model for chemically homogenous evolution.

**Reply by hulin-Lee** [reply]
Thanks again!
